src/cls/IPM/Lifecycle/Base.cls

```diff
@@ -881,15 +881,16 @@ Method %Export(ByRef pParams, ByRef pTargetDirectory As %String, Output pDepende
 				Quit
 			}
 		}
-    Set tParams("ModuleExportPath") = pTargetDirectory
+		Set tParams("ModuleExportPath") = pTargetDirectory
 		
-		Set tSC = ..Module.GetResolvedReferences(.tResourceArray,..#EXPORTDEPENDENCIES,..PhaseList,1,.pDependencyGraph)
+		Set tExportDependencies = $Get(pParams("ExportDependencies"), ..#EXPORTDEPENDENCIES)
```

Comment:
Out of curiosity: why not choose to update the EXPORTDEPENDENCIES parameter continue to use that in the condition below? And if not, is there any reason to keep the EXPORTDEPENDENCIES parameter in the class?
@isc-shuliu

Reply:
I was thinking that we need to maintain backward compatibility. There may be existing published packages that make use of `#EXPORTDEPENDENCIES` parameter already, so we should default back to the value of `#EXPORTDEPENDENCIES`